spec/integrity.yml

```diff
@@ -169,10 +169,13 @@ data/inline_image.pdf:
   :md5: b02bfbb6b0ee7c7d7df1b5f0c1f198c1
 data/inline_image_single_line_content_stream.pdf:
   :bytes: 130786
-  :md5: 096fb28baf29a716066768cb31182b73
+  :md5: '096fb28baf29a716066768cb31182b73'
```

Comment:
as in #256, it'd be good if the diffs avoided unrelated changes like this

Reply:
👍